app/studentApi.js

```diff
@@ -0,0 +1,66 @@
+"use strict"
+
+// dependencies
+const express = require("express");
+var router = express.Router();
+var router = require('./router.js');
+const studentDB = require("./studentDB.json");
```

Comment:
One router is overriding the other:

![image](https://cloud.githubusercontent.com/assets/5324414/21114534/8bb7573c-c094-11e6-9b13-30ca5341a9cb.png)